src/g_doom/a_archvile.cpp

```diff
@@ -152,5 +154,6 @@ DEFINE_ACTION_FUNCTION_PARAMS(AActor, A_VileAttack)
 		P_RadiusAttack (fire, self, blastdmg, blastrad, dmgtype, 0);
 	}
 	if (!(target->flags7 & MF7_DONTTHRUST))
-		target->velz = Scale(thrust, 1000, target->Mass);
+		target->velz = Scale(thrust, 1000, target->Mass) * thrustmul;

```

Comment:
"Thrustmul is already a fixed_t."

That would be why Blzut3 asked you that question, then. When you multiply a fixed_t by another fixed_t, you have to use a FixedMul. In the same way that if you multiply 50% by 50% you shouldn't get 2500% as the result...